_sass/_cayman.scss

```diff
@@ -39,6 +39,10 @@ a {
   &:hover {
     text-decoration: underline;
   }
+
+  &:visited {
+	color: $link-visited-color;
```

Comment:
we should be using spaces in all our source files, not tabs. would you mind fixing this in a follow-up PR?